.github/workflows/check-packaging.yml

```diff
@@ -0,0 +1,20 @@
+name: Check packaging
+on:
+  pull_request:
+  push:
+    branches:
+    - main
+jobs:
+  check-packaging:
+    name: Build and check the distribution package
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Install test dependencies
+      run: |
+        python3 -m pip install --upgrade pip setuptools
+        python3 -m pip install tox-gh-actions
```

Comment:
Out of curiosity: is there a reason for using `tox-gh-actions` instead of `tox`? As far as I can tell you're not really using any of its specific features.

Reply:
Well spotted! Indeed I wanted to test several Python versions with the matrix capabilities at the beginning. But I fell into endless troubles with petsc4py. I prefer to stick to one version for now and come back to the matrix-thing later-on.
Thanks